hceres/src/main/java/org/centrale/hceres/controller/TeamController.java

```diff
@@ -0,0 +1,55 @@
+package org.centrale.hceres.controller;
+
+
+import org.centrale.hceres.items.Activity;
+import org.centrale.hceres.items.Team;
+import org.centrale.hceres.service.TeamService;
+import org.centrale.hceres.util.RequestParseException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.text.ParseException;
```

Comment:
turn on sonar lint to fix code smell.
for example here remove unused import 
import org.centrale.hceres.items.Activity;
import java.text.ParseException;
import java.util.Optional